package.json

```diff
@@ -12,13 +12,14 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "expo": "^35.0.0",
-    "react": "16.8.3",
-    "react-dom": "16.8.3",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
+    "expo": "^37.0.7",
+    "react": "16.9.0",
+    "react-dom": "16.9.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
     "react-proxy": "1.1.8"
   },
   "devDependencies": {
-    "babel-preset-expo": "^5.2.0"
+    "@babel/core": "^7.9.0",
```

Comment:
Why do we need this dependency? Scala.js is still emitting ES5 so not sure why Babel would need to be involved.

Reply:
You are right, is not needed, removed. Can you merge (and squash) like this?